### PR TITLE
tree-wide: change only_for_archs to archs=

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/blender/template [hunk 1]
```diff
@@ -21,7 +21,7 @@ patch_args="-Np1"
 pycompile_version="$py3_ver"
 pycompile_dirs="/usr/share/blender/$version/scripts"
 
-only_for_archs="x86_64 x86_64-musl i686 i686-musl ppc64le ppc64le-musl ppc64-musl"
+archs="x86_64* i686* ppc64le ppc64le-musl ppc64-musl"
```
Comment (Gottox) [inline srcpkgs/blender/template]: @q66 can this be reduced to `x86_64* i686* ppc64*`?

File: srcpkgs/ci20-base/template [hunk 2]
```diff
@@ -7,7 +7,7 @@ short_desc="Void Linux MIPS CI20 Creator base files"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="Public Domain"
 
-only_for_archs="mipsel mipsel-musl mipselhf-musl"
+archs="mipsel mipsel-musl mipselhf-musl"
```
Comment (Gottox) [inline srcpkgs/ci20-base/template]: `mips*` or `mipsel*` here?

File: srcpkgs/ci20-kernel/template [hunk 3]
```diff
@@ -21,7 +21,7 @@ nostrip=yes
 noverifyrdeps=yes
 noshlibprovides=yes
 
-only_for_archs="mipsel mipsel-musl mipselhf-musl"
+archs="mipsel mipsel-musl mipselhf-musl"
```
Comment (Gottox) [inline srcpkgs/ci20-kernel/template]: same

File: srcpkgs/ci20-uboot/template [hunk 4]
```diff
@@ -8,7 +8,7 @@ maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="GPL-2"
 homepage="http://www.denx.de/wiki/U-Boot/WebHome"
 
-only_for_archs="mipsel mipsel-musl mipselhf-musl"
+archs="mipsel mipsel-musl mipselhf-musl"
```
Comment (Gottox) [inline srcpkgs/ci20-uboot/template]: same

File: srcpkgs/docker-credential-pass/template [hunk 5]
```diff
@@ -3,7 +3,7 @@ pkgname=docker-credential-pass
 version=0.6.1
 revision=2
 wrksrc="docker-credential-helpers-${version}"
-only_for_archs="x86_64 x86_64-musl ppc64le ppc64le-musl"
+archs="x86_64* ppc64le ppc64le-musl"
```
Comment (Gottox) [inline srcpkgs/docker-credential-pass/template]: @q66 does `ppc64*` work here?

File: srcpkgs/kodi/template [hunk 6]
```diff
@@ -16,7 +16,7 @@ patch_args="-Np1"
 LDFLAGS+=" -Wl,-z,stack-size=1048576"
 
 lib32disabled=yes
-only_for_archs="i686 i686-musl x86_64 x86_64-musl aarch64 aarch64-musl
+archs="i686* x86_64* aarch64 aarch64-musl
```
Comment (Gottox) [inline srcpkgs/kodi/template]: `aarch64*`

File: srcpkgs/kubernetes/template [hunk 7]
```diff
@@ -4,7 +4,7 @@ version=1.13.3
 revision=1
 build_style=go
 nocross=yes
-only_for_archs="x86_64 x86_64-musl ppc64le ppc64le-musl"
+archs="x86_64* ppc64le ppc64le-musl"
```
Comment (Gottox) [inline srcpkgs/kubernetes/template]: `ppc64le*`?

File: srcpkgs/libglob/template [hunk 8]
```diff
@@ -3,7 +3,7 @@ pkgname=libglob
 version=1.0
 revision=2
 build_style=gnu-makefile
-only_for_archs="aarch64-musl armv6l-musl armv7l-musl i686-musl mips-musl mipshf-musl mipsel-musl mipselhf-musl x86_64-musl ppc64le-musl ppc64-musl"
+archs="aarch64-musl armv6l-musl armv7l-musl i686-musl mips-musl mipshf-musl mipsel-musl mipselhf-musl x86_64-musl ppc64le-musl ppc64-musl"
```
Comment (Gottox) [inline srcpkgs/libglob/template]: `*-musl`

File: srcpkgs/libucontext/template [hunk 9]
```diff
@@ -7,7 +7,7 @@ maintainer="Peter Bui <pbui@github.bx612.space>"
 license="ISC"
 homepage="https://github.com/kaniini/libucontext"
 distfiles="https://distfiles.dereferenced.org/${pkgname}/${pkgname}-${version}.tar.xz"
-only_for_archs="aarch64-musl armv5tel-musl armv6l-musl armv7l-musl i686-musl mips-musl mipshf-musl mipsel-musl mipselhf-musl x86_64-musl ppc64le-musl ppc64-musl"
+archs="aarch64-musl armv5tel-musl armv6l-musl armv7l-musl i686-musl mips-musl mipshf-musl mipsel-musl mipselhf-musl x86_64-musl ppc64le-musl ppc64-musl"
```
Comment (Gottox) [inline srcpkgs/libucontext/template]: `*-musl`

File: srcpkgs/blender/template (hunk 1 above)
Comment (q66) [reply srcpkgs/blender/template]: Go ahead.

File: srcpkgs/docker-credential-pass/template (hunk 5 above)
Comment (q66) [reply srcpkgs/docker-credential-pass/template]: Should be `ppc64le*`

File: srcpkgs/kubernetes/template (hunk 7 above)
Comment (q66) [reply srcpkgs/kubernetes/template]: Yep

**Comment by maxice8** [comment]
Applied a few more seds

another round of review

**Pull request merged**